frontend/package.json

```diff
@@ -125,6 +127,7 @@
     "enzyme": "3.11.0",
     "enzyme-to-json": "3.4.4",
     "husky": "1.3.1",
+    "if-node-version": "^1.1.1",
     "jest-canvas-mock": "2.2.0",
```

Comment:
We don't need if-node-version, we are goign to force to use 18 or greater and flag is supported.

Redo scripts-tasks